package.json

```diff
@@ -23,7 +23,8 @@
     "generate": "stencil generate",
     "storybook": "start-storybook -p 6006",
     "serve": "stencil build --dev --watch --serve",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "prepare": "npm run build"
```

Comment:
Are we going to have problems here using `npm` instead of `yarn`?

Reply:
No, I don't think so. This just runs the script. :man_shrugging:

Reply:
I think I was remembering that `npm link` that we were working with and how it _insisted_ on doing an `npm install` :sweat_smile: